pygotham/frontend/talks.py

```diff
@@ -118,7 +118,7 @@ def proposal(pk=None):
 @route(blueprint, '/schedule')
 def schedule():
     event = g.current_event
-    if not event.schedule_is_published:
+    if not event.schedule_is_published and not current_user.has_role('admin'):
```

Comment:
In general I try not to repeat `not`s.

    if not (event.schedule_is_published or current_user.has_role('admin'):

It's also a tad shorter.